### PR TITLE
#39 - Add drink recommendations 

### PR DESCRIPTION
**Pull request by jbgerth**

### CODE REVIEW

File: src/commands/cereal/drink.ts [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+import { Drink } from "../../models/drink";
+import { Ingredient } from "../../models/ingredient";
+
+const dispatch = async (args: string[]): Promise<Drink | void> => {
+  const baseUrl = `${COCKTAIL.BASE_URL}${COCKTAIL.API_VERSION}${COCKTAIL.API_KEY}`;
+  let drink_url = baseUrl;
+  if (args.length === 0) {
+    drink_url += COCKTAIL.RANDOM_URL;
+  } else if (args[0].length === 1) {
+    const search_for_letter: string = args[0];
```
Comment (coinzz) [inline src/commands/cereal/drink.ts:15]: Also _.toLower() required?

File: tests/commands.cereal.drink.test.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+});
+
+const empty_drinks = { drinks: [] };
+it("reject promise, because of empty array", async () => {
+  expect.assertions(1);
+  try {
+    const result = await selectDrinkFromList(empty_drinks);
+    console.log(result);
+  } catch (e) {
+    expect(e.message).toEqual("No drink was found");
```
Comment (coinzz) [inline tests/commands.cereal.drink.test.js:105]: Changes to the Error-text will fail this test. Maybe use a global variable in drink.ts?

File: tests/commands.cereal.drink.test.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+});
+
+const undefined_drinks = undefined;
+it("reject promise, because of undefined", async () => {
+  expect.assertions(1);
+  try {
+    const result = await selectDrinkFromList(undefined_drinks);
+    console.log(result);
+  } catch (e) {
+    expect(e.message).toEqual("No drink was found");
```
Comment (coinzz) [inline tests/commands.cereal.drink.test.js:116]: Changes to the Error-text will fail this test. Maybe use a global variable in drink.ts?

**Review by coinzz — CHANGES_REQUESTED** [review]

File: src/commands/cereal/drink.ts (hunk 1 above)
Comment (jbgerth) [reply src/commands/cereal/drink.ts:15]: See #47

**Review by coinzz — APPROVED** [review]

**Pull request merged**